libs/backend/src/cast_vote_records/export.ts

```diff
@@ -165,12 +178,10 @@ async function getExportDirectoryPathRelativeToUsbMountPoint(
     generateElectionBasedSubfolderName(election, electionHash),
     exportDirectoryName
   );
-  if (exportDirectoryNeedsCreation) {
-    await fs.mkdir(
-      path.join(usbMountPoint, exportDirectoryPathRelativeToUsbMountPoint),
-      { recursive: true }
-    );
-  }
+  await fs.mkdir(
+    path.join(usbMountPoint, exportDirectoryPathRelativeToUsbMountPoint),
+    { recursive: true }
+  );
```

Comment:
Minor change to existing logic. Rather than only calling `fs.mkdir` when we believe it to be necessary, I'm now always calling it. The operation is inexpensive and idempotent, and after the addition of resync logic, the previous code was actually missing cases where export directory creation was necessary.